chia/_tests/core/test_merkle_set.py

```diff
@@ -358,15 +331,14 @@ def test_validate_additions_full_list(num_coins: int, batch_size: int, seeded_ra
     # the root can be computed by all the removals
     coins = make_test_coins(num_coins, seeded_random)
 
-    additions_merkle_set = MerkleSet()
     additions: List[Tuple[bytes32, List[Coin]]] = []
+    leafs: List[bytes32] = []
     for coin_batch in to_batches(coins, batch_size):
         puzzle_hash = bytes32.random(seeded_random)
         additions.append((puzzle_hash, coin_batch.entries))
-        additions_merkle_set.add_already_hashed(puzzle_hash)
-        additions_merkle_set.add_already_hashed(
-            std_hash(b"".join(sorted([coin.name() for coin in coin_batch.entries], reverse=True)))
-        )
+        leafs.append(puzzle_hash)
+        leafs.append(std_hash(b"".join(sorted([coin.name() for coin in coin_batch.entries], reverse=True))))
```

Comment:
```suggestion
        leafs.append(std_hash(b"".join(sorted((coin.name() for coin in coin_batch.entries), reverse=True))))
```

Reply:
are '(`, `)` better than `[`, `]`?

Reply:
Mmm, the `()` might actually be optional here.  Either way, it's then a generator instead of a list.  As such it doesn't actually create a list.  Now yeah, I think that sorted will create a list to sort in but I think it would do that regardless so this still results in one less list being built.  Is it a huge deal?  Without checking the usages and so on, I would guess it is not a big deal.  Just trying to encourage habits around not creating lists when they are not being used as lists.